docs/section5/Rule5-15.md

```diff
@@ -0,0 +1,57 @@
+**Appendix G Section Reference:** Tables G3.4-1 to G3.4-8  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:** None  
+
+**Manual Check:** Yes  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** Tables G3.4-1 to G3.4-8  
+**Function Call:**  
+
+  1. get_surface_conditioning_category()  
+  2. get_opaque_surface_type()  
+
+## Rule Logic:  
+
+- Get building climate zone: ```climate_zone = B_RMR.weather.climate_zone```  
+
+- Get surface conditioning category dictionary for B_RMR: ```scc_dictionary_b = get_surface_conditioning_category(B_RMR)```  
+
+- For each building segment in the Baseline model: ```for building_segment_b in B_RMR.building.building_segments:```  
+
+  - For each thermal_block in building segment: ```for thermal_block_b in building_segment_b.thermal_blocks:```  
+
+    - For each zone in thermal block: ```for zone_b in thermal_block_b.zones:```  
+
+      - For each surface in zone: ```for surface_b in zone_b.surfaces:```  
+
+        - Check if surface is unheated slab-on-grade: ```if get_opaque_surface_type(surface_b) == "UNHEATED SLAB-ON-GRADE":```  
+
+          - Get surface construction: ```surface_construction_b = surface_b.construction```  
+
+          - Get surface conditioning category: ```scc_b = scc_dictionary_b[surface_b.id]```  
+
+            - If surface is exterior residential, exterior non-residential, or semi-exterior, get baseline construction from Table G3.4-1 to G3.4-8 based on climate zone, surface conditioning category and surface type: ```if ( ( scc_b == "EXTERIOR RESIDENTIAL" ) OR ( scc_b == "EXTERIOR NON-RESIDENTIAL" ) OR ( scc_b == "SEMI-EXTERIOR" ) ): target_f_factor = data_lookup(table_G3_4, climate_zone, scc_b, "SLAB-ON-GRADE FLOOR")```  
+
+            - Else if surface is exterior mixed, get baseline construction for both residential and non-residential type slab-on-grade floor: ```else if ( scc_b == "EXTERIOR MIXED" ): target_f_factor_res = data_lookup(table_G3_4, climate_zone, "EXTERIOR RESIDENTIAL", "SLAB-ON-GRADE FLOOR"), target_f_factor_nonres = data_lookup(table_G3_4, climate_zone, "EXTERIOR NON-RESIDENTIAL", "SLAB-ON-GRADE FLOOR")```  
+
+              - If residential and non-residential type slab-on-grade floor construction requirements are the same, save as baseline construction: ```if target_f_factor_res == target_f_factor_nonres: target_f_factor = target_f_factor_res```  
+
+              - Else: ```manual_review_flag = TRUE```  
+
+          **Rule Assertion:**  
+
+          Case 1: If zone has both residential and non-residential spaces and the construction requirements for slab-on-grade floor are different, request manual review: ```if manual_review_flag == TRUE: RAISE_WARNING```  
+
+          Case 2: Else if slab-on-grade floor F-factor matches Table G3.4: ```else if surface_construction_b.f_factor == target_f_factor: PASS```  
+
+          Case 3: Else: ```else: FAIL```  
+
```

Comment:
Based on Section 3 definition, "slab-on-grade floor: that portion of a slab floor of the building envelope that is in contact
with the ground and that is either above grade or is less than or equal to 24 in. below the final elevation of the nearest exterior grade.". So if floor is more than 24" below grade it will be unregulated, but there is not way for RCT to determine that. So perhaps Case 3 outcome should be caution instead of fail?

Reply:
Modified.